src/components/Contacts/ContactsHeader/ContactsHeader.test.tsx

```diff
@@ -195,12 +195,13 @@ describe('ContactFilters', () => {
         />
       </ThemeProvider>,
     );
-
     const textbox = getByRole('textbox');
 
     userEvent.type(textbox, searchText);
 
     expect(toggleFilterPanel).not.toHaveBeenCalled();
-    expect(onSearchTermChanged).toHaveBeenCalledWith(searchText);
+    setTimeout(() => {
+      expect(onSearchTermChanged).toHaveBeenCalledWith(searchText);
+    }, 500);
```

Comment:
Idk if this is the right way to test it. The test will complete successfully before the timeout is fired so that expect is probably not called until after the test finishes.

Reply:
gotcha... I'll try to see if there's a better way to wait for the debounce to finish before testing this.